base/io.jl

```diff
@@ -125,6 +125,9 @@ function write(s::IO, a::AbstractArray)
     end
     return nb
 end
+"""Write directly to a named file. Equivalent to `open(io->write(io,x), filename, "w")`."""
```

Comment:
agreed the formatting should have the signature, was doing it quickly on a phone